src/pybind/mgr/cephadm/tests/test_service_discovery.py

```diff
@@ -88,11 +88,11 @@ def get_module_option_ex(self, module, option, default_value):
         return "9283"
 
 
-class TestCephadmService:
+class TestServiceDiscovery:
 
     def test_get_sd_config_prometheus(self):
         mgr = FakeMgr()
-        root = Root(mgr)
+        root = Root(mgr, '', '', '', '')
```

Comment:
The second param should be a port, so a number, right? What about adding defaults, so this doesn't end up being so repetitive?

Reply:
Fixed by using defaults